bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -691,6 +691,13 @@ Bridge netatmo:account:myaccount "Netatmo Account" [clientId="xxxxx", clientSecr
                 Type live-stream-url : live#vpn-stream-url [ quality="low" ]
         }
     }
+    Bridge home myhomeheating "Home heating" [ id="...." ] {
```

Comment:
Minor, but for consistency with other examples, we would need some pseudo id ("xx:00:00:aa:aa:aa"). Even if not, perhaps consistency with the next lines:
```suggestion
    Bridge home myhomeheating "Home heating" [ id="..." ] {
```